src/main/java/mate/controller/car/CreateCarController.java

```diff
@@ -0,0 +1,41 @@
+package mate.controller.car;
+
+import java.io.IOException;
+import java.util.Collections;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Manufacturer;
+import mate.service.CarService;
+import mate.service.ManufacturerService;
+
+@WebServlet(urlPatterns = "/cars/add")
+public class CreateCarController extends HttpServlet {

```

Comment:
if use create verb in the name of the controller and jsp file, then use it in URL too, so naming was consistent, the same for other classes and jsps

Reply:
✅